test/ruby/test_regexp.rb

```diff
@@ -646,13 +646,37 @@ def test_initialize
     assert_equal(/foo/, assert_no_warning(/ignored/) {Regexp.new(/foo/)})
     assert_equal(/foo/, assert_no_warning(/ignored/) {Regexp.new(/foo/, timeout: nil)})
 
-    assert_equal(Encoding.find("US-ASCII"), Regexp.new("b..", nil, "n").encoding)
-    assert_equal("bar", "foobarbaz"[Regexp.new("b..", nil, "n")])
-    assert_equal(//n, Regexp.new("", nil, "n"))
-
     arg_encoding_none = 32 # ARG_ENCODING_NONE is implementation defined value
```

Comment:
```suggestion
    arg_encoding_none = //n.options # ARG_ENCODING_NONE is implementation defined value
```

Reply:
Thanks, I committed this suggestion when I rebased.